BitWatch/HomeViewController.swift

```diff
@@ -9,30 +9,18 @@
 import UIKit
 import BitWatchKit
 
-class HomeViewController: UIViewController {
 
+class HomeViewController: UIViewController, HomeVCDelegate{
```

Comment:
Need to remove HomeVCDelegate. Will do that after pullrequest. Wasn't sure how to do blocks but then I figured it out.